### ISSUE TITLE
ImportError: No module named 'src'

### ISSUE THREAD

**Issue by Madic-** [dr1s/pihole_exporter.py#2]
On a standard Ubuntu installation I get the following error:
ImportError: No module named 'src'

On default there isn't python3 installed so I needed to install it via apt install python3-pip.
Than I tried to install your exporter via pip3 install git+https://github.com/dr1s/pihole_exporter.py.git
and got the above error as soon as I want to start it

**Comment by Madic-** [comment]
After installing json2xml via pip3 the "src.pihole_exporter" module can not be found

**Comment by dr1s** [comment]
Thank you for the report!

I just pushed a commit to master, please try again.

**Comment by Madic-** [comment]
Yes, now it's working. Thank your. But the command in the readme is wrong.
It's 'pip3 install --upgrade git+https://github.com/dr1s/pihole_exporter.py.git'

Also thank you for the exporter. It's working great
